packages/frontend/src/components/cookie-policy/CookiePolicy.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import cookiePolicyPdf from '../../documents/cookie-policy.pdf';
+import { Cookie } from '../../utils/cookies';
+import { StyledButton } from '../common/Button';
+
+// Copypasted close icon from modal/CloseButton.js
+const CloseIcon = () => (<svg width="35" height="35" viewBox="0 0 35 35" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think it would be cleaner and more future proof to create a `CloseIcon.js` file inside `packages/frontend/src/components/svg/` and import from that folder.